test_launch_ros/test/test_launch_ros/actions/test_ros_timer_action.py

```diff
@@ -0,0 +1,201 @@
+# Copyright 2021 Open Source Robotics Foundation, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+"""Tests for the RosTimerAction Action."""
+import sys
+import time
+import threading
+from functools import partial
+
+import launch
+import launch.event_handlers
+
+from launch_ros.actions import RosTimerAction
+
+import rclpy
+from rclpy.clock import Clock, ClockType
+
+from rosgraph_msgs.msg import Clock as ClockMsg
+from builtin_interfaces.msg import Time
+
+
+def test_multiple_launch_with_timers():
+    def generate_launch_description():
+        return launch.LaunchDescription([
+
+            launch.actions.ExecuteProcess(
+                cmd=[sys.executable, '-c', 'while True: pass'],
+            ),
```

Comment:
minor:
It's not immediately clear why we need this; I suppose this was copied over from `test_timer_action.py`. I'm pretty sure it's here to keep the launch process alive. Since we create the same action throughout the tests, I suggest refactoring into a semantically meaningful class or function in this file (e.g. "KeepAlive"). Note, we have a similar utility in `launch_testing` (though I wouldn't depend on it here): https://github.com/ros2/launch/blob/3a5c9df2e384271637ac029f48f020b7916498d6/launch_testing/launch_testing/util/__init__.py#L24

Reply:
I've removed this as it doesn't seem to be necessary for the tests.